src/collective/cover/tests/test_locked_cover.robot

```diff
@@ -43,6 +43,7 @@ Test Locked Cover
     Enable Autologin as  Site Administrator
     Goto Homepage
     Click Link  link=My Cover
+    Wait Until Page Contains  Locked
```

Comment:
why note replace the whole thing?

```diff
-Page Should Contain  Locked  ${LOCKED_MESSAGE}
+Wait Until Page Contains  Locked  ${LOCKED_MESSAGE}
```

Reply:
```Wait Until Page Contains``` doen't support a second term for comparison.

I'll only use ${LOCKED_MESSAGE}

Reply:
Done.